libraries/AP_Mount/AP_Mount.h

```diff
@@ -74,17 +74,17 @@ class AP_Mount
     }
 
     // Enums
-    enum MountType {
-        Mount_Type_None = 0,            /// no mount
-        Mount_Type_Servo = 1,           /// servo controlled mount
-        Mount_Type_SoloGimbal = 2,      /// Solo's gimbal
-        Mount_Type_Alexmos = 3,         /// Alexmos mount
-        Mount_Type_SToRM32 = 4,         /// SToRM32 mount using MAVLink protocol
-        Mount_Type_SToRM32_serial = 5,  /// SToRM32 mount using custom serial protocol
-        Mount_Type_Gremsy = 6,          /// Gremsy gimbal using MAVLink v2 Gimbal protocol
-        Mount_Type_BrushlessPWM = 7,    /// Brushless (stabilized) gimbal using PWM protocol
-        Mount_Type_Siyi = 8,            /// Siyi gimbal using custom serial protocol
-        Mount_Type_Scripting = 9,       /// Scripting gimbal driver
+    enum class Type {
```

Comment:
@peterbarker,

I'm not a huge fan of using the word "Type" even though I guess it's not a reserved word in C++.

Reply:
GPS::Type
PrecLand::Type
Rangefinder::Type
OpticalFlow::Type
TemperatureSensor::Type
AP_BattMonitor::Type
AP_EFI::Type
AP_ADSB::Type
AP_Generator::Type

those are the ones I could find quickly.

git grep "enum.*class.*\wType" does find a few - like `enum class CameraType` and AP_Logger::Backend_Type - but there's not many perhaps 4 or 5.  It's only the `enum class`es that matter as without that the language doesn't guarantee your namespacing.